### DISCUSSION TITLE
SSIkit EVM trust registry

### DISCUSSION THREAD

**Discussion by gftwally** [walt-id/.github#61] [Help] [answered]
Hi All

From documentation I can see that SSI Kit supports EVM registry

_Our products are agnostic towards the underlying technologies used to implement Trust Registries, which means that the SSI Kit is potentially compatible with any type of Trust Registry.
The SSI Kit supports:
Permissionless Blockchains (e.g. Ethereum),
Permissioned Blockchains (e.g. Ethereum Enterprise/Hyperledger Besu),_

I have no clue how to interact for ex with BESU. Is there any repo with some smart contracts to maintain the trust repository?

Please advise.

**Comment by taminobaumann** [comment, marked as answer]
Hey @gftwally, 

thanks for reaching out! 

Yes, it's correct that the SSI-Kit has the capabilities to interact with external Trust Registries. Though at this point, we haven't written an implementation for Ethereum yet. However, you still have the option to extend the SSI-Kit and write an extension that can connect to the external Trust Registry of your choice.

Best
Tamino
